docs/src/install/core.rst

```diff
@@ -1,32 +1,38 @@
-****
-Core
-****
+************************
+Installation of the Core
```

Comment:
I would go with another sentencing, more along the lines of `Installation of Orion's core` or something like that.